packages/@aws-cdk/pipelines/lib/private/template-configuration.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'fs';
```

Comment:
Should this be moved under the `codepipeline` directory?

Reply:
Used by both implementations. Preferably not.